src/authn-component/index.jsx

```diff
@@ -130,6 +130,7 @@ export const AuthnComponent = ({
       close={close}
       hasCloseButton={hasCloseButton}
       size={screenSize}
+      isFullScreen={screenSize === 'fullscreen'}
```

Comment:
Why do we need to pass this prop, We are already passing `size` prop and we can add this condition inside the `BaseContainer` component.

Reply:
yes you are right we can do that